apps/api/lib/api/web/controllers/friend_request_controller.ex

```diff
@@ -30,7 +30,8 @@ defmodule API.Web.FriendRequestController do
       API.FriendRequest.delete(id)
     do
       conn
-      |> put_status(200)
+      |> put_status(204)
```

Comment:
maybe we should decide once and for all on atoms vs ints for status codes, for consistency

Reply:
Agreed, I'm fine with either one. Im just always to lazy to look up the atom haha